app/src/interfaces/_system/system-fields/system-fields.vue

```diff
@@ -21,13 +21,20 @@
 			</draggable>
 		</v-list>
 		<v-menu placement="bottom-start" show-arrow>
+			<button class="selectAllBtn" @click="selectAll">
+				<v-icon name="check_box" />
+				{{ t('select_all') }}
+			</button>
+			<button class="selectAllBtn" @click="deselectAll">
+				<v-icon name="check_box_outline_blank" />
+				{{ t('deselect_all') }}
+			</button>
```

Comment:
Instead of separately styled buttons, lets rely on a v-list here to make that dropdown styled as normal. I believe v-field-list similarly relies on v-list and it's sub-components